app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>BudgetOnLine</title>
+    <title>MoneyChecker</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
```

Comment:
- [ ] Great job so far but you should add unit tests for all the most important components and integration tests for all the most important components of your application. Kindly address this ✍🏻